support/ccm.rb

```diff
@@ -714,6 +714,17 @@ def change_tombstone_thresholds
       start
     end
 
+    def enable_triggers
+      trigger_root = File.expand_path(File.dirname(__FILE__) + '/../support/triggers')
+      ccm_node_conf_dir = "~/.ccm/#{@cluster.name}"
+
+      @cluster.hosts.each_index do |n|
+        `mkdir #{ccm_node_conf_dir}/node#{n+1}/conf/triggers`
+        `cp #{trigger_root}/AuditTrigger.properties #{ccm_node_conf_dir}/node#{n+1}/conf`
```

Comment:
Interesting; I didn't copy AuditTrigger.properties in my experiments and it worked fine. I bet the properties file matters when you actually trigger the trigger. If you feel like it, you can remove the AuditTrigger.properties from this, and the extra files involved in these tests will be just the jar. I don't feel strongly one way or the other.